### ISSUE TITLE
Can't sync gradle on a fresh project with the plugin added (KotlinJvmAndroidCompilation with name 'debug' not found.)

### ISSUE THREAD

**Issue by ddudek** [dropbox/dropshots#19]
👋 

I wanted to try out this library, but as soon as I add plugin:
```
plugins {
    id("com.android.application")
    id 'org.jetbrains.kotlin.android'
    id("com.dropbox.dropshots") version "0.2.0"
}
```
(plus `mavenCentral()` in the repos, I already had it) the project can't sync gradle with an error:
```
A problem occurred configuring project ':app'.
> Failed to notify project evaluation listener.
   > org/gradle/configurationcache/extensions/CharSequenceExtensionsKt
   > KotlinJvmAndroidCompilation with name 'debug' not found.
```
<img width="1466" alt="Zrzut ekranu 2022-06-30 o 12 29 21" src="https://user-images.githubusercontent.com/475488/176701670-6957850b-1b5c-4336-84e9-e001cf2945a6.png">

Tried and can't make it run in three projects:
- my main work project
- fresh sample project
- fresh sample project at my colleague's

with the same result

Am I missing something? 🤔 

Environment:
a) macbook with m1 pro chip, Android Studio Chipmunk | 2021.2.1 Patch 1
b) my colleague's macbook with intel

Thanks for new lightweight library, sounds like something that I'd need 👏

**Comment by ddudek** [comment]
[SampleDropshot.zip](https://github.com/dropbox/dropshots/files/9026984/SampleDropshot.zip)
attached above
It's fresh clean project in AS, and with issue reproduced

**Comment by ddudek** [comment]
thanks for the fix 👏  is the 0.3.0 version already available? AS can't find it, and doesn't seem to be available on maven: https://repo.maven.apache.org/maven2/com/dropbox/dropshots/dropshots/
